TicketSwap/solar
# feat(Icons): Add Tidal icon

# Description

See title.

## How to test

- Checkout
- Start Storybook
- Go to the icons section
- Verify the Tidal icon is working as expected
